auth-web/src/views/auth/TermsOfServiceView.vue

```diff
@@ -89,6 +89,10 @@ export default class TermsOfServiceView extends Mixins(NextPageMixin) {
     this.atBottom = (e.target.scrollHeight - e.target.scrollTop) <= (e.target.offsetHeight + 25)
   }
 
+  mounted () {
+    this.$store.commit('updateHeader')
```

Comment:
Shouldn't need to do this, is something not updating?

Reply:
Its needed, the account was not updating without it

Reply:
Okay - I may try to find a better way to do this.  Forcing a header remount is expensive.